soc/xtensa/intel_adsp/common/include/cavs_test.h

```diff
@@ -0,0 +1,62 @@
+/* Copyright (c) 2022 Intel Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Nitpick: the motion of this header wants to be a separate patch.  Also this is really heavily coupled to the cavstool.py unit test script, so maybe a "cavstool.h" name or something might make it clearer what belongs here.  It's not just generic stuff for "testing".

Reply:
Done